package.json

```diff
@@ -62,10 +62,10 @@
     "husky": "4.2.5",
     "jest": "25.5.4",
     "lint-staged": "12.5.0",
-    "prettier": "1.19.1"
+    "prettier": "2.7.1"
   },
   "engines": {
-    "node": "12.22.x",
+    "node": ">=16.15",
     "yarn": "1.22.19"
```

Comment:
Will this roll to 18 when that happens?

Reply:
The version should stay as 16 in the Vercel settings, but wouldn't break if it went up. I'll look at if there's a benefit to it being more specific next time.